app/Http/Controllers/Api/v1/EntityMoveApiController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Http\Controllers\Api\v1;
+
+use App\Models\Campaign;
+use App\Models\Entity;
+use App\Http\Requests\MoveEntity as Request;
+use App\Services\Entity\MoveService;
+
+class EntityMoveApiController extends ApiController
+{
+    protected MoveService $service;
+
+    public function __construct(MoveService $service)
+    {
+        $this->middleware('auth');
+
+        $this->service = $service;
+    }
+
+    /**
+     * @throws \Illuminate\Auth\Access\AuthorizationException
+     */
+    public function transfer(Request $request, Campaign $campaign)
+    {
+        $this->authorize('access', $campaign);
+        $count = 0;
+        $copy = null === $request->copy ? true : $request->copy;
+
+        foreach ($request->entities as $id) {
+            $entity = Entity::find($id);
+            if ($this->authorize('update', $entity->child)) {
+                $this->service
```

Comment:
Ugly error when trying to copy to a campaign I'm not a part of. Missing a try/catch with a TranslatableException catch